docs/import_vulnerability_data.md

```diff
@@ -0,0 +1,58 @@
+    Jul 3 20:40:46 172.18.0.1 clair[3516]: {"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-04 03:40:46.768924","updater name":"alpine"}
+    Jul 3 20:40:47 172.18.0.1 clair[3516]: {"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-04 03:40:47.190982","updater name":"oracle"}
+    Jul 3 20:41:07 172.18.0.1 clair[3516]: {"Event":"Debian buster is not mapped to any version number (eg. Jessie-\u003e8). Please update me.","Level":"warning","Location":"debian.go:128","Time":"2017-07-04 03:41:07.833720"}
+    Jul 3 20:41:07 172.18.0.1 clair[3516]: {"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-04 03:41:07.833975","updater name":"debian"}
+    Jul 4 00:26:17 172.18.0.1 clair[3516]: {"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-04 07:26:17.596986","updater name":"ubuntu"}
+    Jul 4 00:26:18 172.18.0.1 clair[3516]: {"Event":"adding metadata to vulnerabilities","Level":"info","Location":"updater.go:253","Time":"2017-07-04 07:26:18.060810"}
+    Jul 4 00:38:05 172.18.0.1 clair[3516]: {"Event":"update finished","Level":"info","Location":"updater.go:198","Time":"2017-07-04 07:38:05.251580"}
+    ```
+- The phrase "finished fetching" indicates that Clair has finished a round of vulnerability update from an endpoint. Please make sure all five endpoints (rhel, alpine, oracle, debian, ubuntu) are updated correctly. 
+
+### Dumping vulnerability data 
+
+- Log in to the host (that is connected to Internet) where Clair database (Postgres) is running.
+- Dump Clair's vulnerability database by the following commands, two files (`vulnerability.sql` and `clear.sql`) are generated:
+
+```
+  $ docker exec clair-db /bin/bash -c  "pg_dump -U postgres -a -t feature -t keyvalue -t namespace -t schema_migrations -t vulnerability -t vulnerability_fixedin_feature" > vulnerability.sql
+  $ docker exec clair-db /bin/bash -c "pg_dump -U postgres -c -s" > clear.sql
+```
+
+### Back up Harbor's Clair database
+Before importing the data, it is strongly recommended to back up the Clair database in Harbor.
+```
+ $ docker exec clair-db /bin/bash -c  "pg_dump -U postgres -c" > all.sql
+```
+
+### Update Harbor's Clair database
+Copy the `vulnerability.sql` and `clear.sql` to the host where Harbor is running on. Run the below commands to import the data to Harbor's Clair database:
+
+```
+ $ docker exec -i clair-db psql -U postgres < clear.sql
+ $ docker exec -i clair-db psql –U postgres < vulnerability.sql
+```
+
+### Rescanning images
+After importing the data, trigger the "Scan All" process in the administrator's Web UI. It will scan all the images automatically using the updated data. Harbor reflects the new changes after the scanning is completed. (Otherwise the newly imported vulnerability data will not take effect.) 
```

Comment:
What about change to this?
After importing the data, trigger the Administration->Configuration->Vulnerability->SCAN NOW in the administrator's Web UI. Harbor reflects the new changes after the scanning is completed. 
(Otherwise the summary of the image vulnerabilities will not display correctly. )